app/migrations/versions/bde1aa0ecc9c_initial_migration.py

```diff
@@ -0,0 +1,38 @@
+"""Initial migration.
```

Comment:
duplicate migration. as a general principle target one migration per PR

Reply:
removed in next version

Reply:
still not fixed